### ISSUE TITLE
Minified React error #61

### ISSUE THREAD

**Issue by Funasitien** [plan-player-analytics/Plan#3250]
### Describe the issue
I got this error with Player > Plugin page
![image](https://github.com/plan-player-analytics/Plan/assets/86372093/66a126db-af7a-4d5d-ae17-de88a3e95142)

### Exceptions & Other Logs
No log in Plan Folder

```
Minified React error #61; visit https://reactjs.org/docs/error-decoder.html?invariant=61 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

### Plugin versions
5.5-build-2461

### Additional information
The plugin page of the user of other servers work fine

**Comment by AuroraLS3** [comment]
This should be fixed in 5.6

**Comment by AuroraLS3** [comment]
If you'd like to test it out it can be downloaded here https://github.com/plan-player-analytics/Plan/releases/tag/5.6.2574

**Comment by AuroraLS3** [comment]
Never mind I found the mistake

**Comment by AuroraLS3** [comment]
Fixed 5.6 build will appear here in 20 minutes <https://github.com/plan-player-analytics/Plan/actions/runs/6439505885>. 5.6 has some changes that make it not possible to downgrade back to 5.5 so I recommend reading the change log above if you decide to try it
